src/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -117,6 +119,8 @@
 		public static Climber climber;
 		public static MotionController chassisTurnMC;
 		public static AligningCamera alignCamera;
+		public static EnableableModifier rightWheelAccelerationCap;
```

Comment:
I don't love having non-material things here, but I can't think of a better place to keep these.